### PR TITLE
Updating models that depend on history

### PR DESCRIPTION
**Pull request by kushagra414**
#1218 Please Review.

### CODE REVIEW

File: bugbug/models/qaneeded.py [hunk 1]
```diff
@@ -79,15 +79,24 @@ def get_labels(self):
...
+
+            if not found_qa:
+                for entry in bug_data["history"]:
+                    for change in entry["changes"]:
+                        if any(
+                            change["added"].startswith(label)
+                            for label in ["qawanted", "qe-verify", "qaurgent"]
+                        ):
+                            classes[bug_id] = 1
+                if bug_id not in classes:
```
Comment (marco-c) [inline bugbug/models/qaneeded.py]: This should be moved outside the `if not found_qa`

Comment (kushagra414) [reply bugbug/models/qaneeded.py]: Done

**Comment by kushagra414** [comment]
@marco-c I did what you said. But still, the test failed?

**Comment by marco-c** [comment]
Check the logs of the test job to see what the problem is, or see which test is failing and run it locally on your machine.

**Comment by kushagra414** [comment]
All Tests have passed.
Review, please

**Comment by marco-c** [comment]
Thanks!

**Pull request merged**